examples/commandShowcase.lean

```diff
@@ -0,0 +1,35 @@
+import Qq
+
+open Qq
+
+-- using the goal type (interpret 2, 3 as Int)
+def f (x : Prop) [Decidable x] : Int :=
+  by_elabq
+    Lean.logInfo x
+    Lean.logInfo x.ty
+    return q(if $x then 2 else 3)
+
+#print f
```

Comment:
Can you use `guard_msgs` to make this a test?

Reply:
Let's just delete this line, if it's not being tested. I think mathlib moved away from noisy tests, and we should do the same here.

Reply:
Ok, should I also rename the file from Showcase to Test? Originally, I intended to use it to show how `run_tacq` can be used but now it is more steered to be mostly a sanity check.